Add dockerhub credentials (#23)
europace/docker-publish-gradle-plugin: README.md

        organisation.set("my-dockerhub-organisation")
    }

## Defaults

> * As there is a publishImage task automatically hooked up into the publishing cycle, the plugin `maven-publish` is
> applied within this plugin.

Reviewer: ```suggestion
* The `publishImage` task is configured to run after Gradle's `publish` task, so this plugin implicitly applies the `maven-publish` plugin.
```

> * As we currently just have docker images in docker hub, this plugin automatically sets the credentials to be able to publish there.

Reviewer: ```suggestion
* Authentication to the default registry (Docker Hub) is enabled by default.
```
